packages/trader/src/Modules/CFD/Containers/cfd-password-modal.tsx

```diff
@@ -27,18 +34,133 @@ import {
+    error_message: string;
+    validatePassword: (values: TCFDPasswordFormValues) => FormikErrors<TCFDPasswordFormValues>;
+};
+
+type TCFDCreatePasswordProps = TCFDPasswordFormReusedProps & {
+    password: string;
+    onSubmit: (values: TCFDPasswordFormValues, actions: FormikActions<TCFDPasswordFormValues>) => void;
+};
+
+type TCFDCreatePasswordFormProps = TCFDPasswordFormReusedProps & {
+    has_mt5_account: boolean;
+    submitPassword: (values: TCFDPasswordFormValues, actions: FormikActions<TCFDPasswordFormValues>) => void;
+};
+
+type TMultiStepRefProps = {
+    goNextStep: () => void;
+    goPrevStep: () => void;
+};
+
+type TCFDPasswordFormProps = TCFDPasswordFormReusedProps & {
+    account_title: string;
+    closeModal: () => void;
+    error_type?: string;
+    form_error?: string;
+    has_mt5_account: boolean;
+    is_bvi: boolean;
+    is_dxtrade_allowed: boolean;
+    is_real_financial_stp: boolean;
+    onCancel: () => void;
+    onForgotPassword: () => void;
+    should_set_trading_password: boolean;
+    should_show_server_form: boolean;
+    submitPassword: (values: TCFDPasswordFormValues, actions: FormikActions<TCFDPasswordFormValues>) => void;
+};
+
+type TCFDServerModalWarningProps = {
+    show_warning: boolean;
+    platform: string;
+};
+
+type TCFDServerFormProps = {
+    mt5_trading_servers: DetailsOfEachServer[];
+    mt5_login_list: TExtendedDetailsOfEachMT5Loginid[];
+    account_title: string;
+    closeModal: () => void;
+    form_error?: string;
+    submitMt5Server: (server: string) => void;
+    platform: string;
+    has_error: boolean;
+    validateServer?: (values: TCFDServerFormValues) => FormikErrors<TCFDServerFormValues>;
+};
+
+type TCFDPasswordModalProps = RouteComponentProps & {
+    account_title: string;
+    account_type: {
+        category?: string;
+        type?: string;
+    };
+    account_status: GetAccountStatus;
+    disableCFDPasswordModal: () => void;
+    email: string;
+    error_message: string;
+    error_type?: string;
+    form_error?: string;
+    getAccountStatus: (platform: string) => void;
+    is_eu: boolean;
+    is_eu_country: boolean;
+    is_fully_authenticated: boolean;
+    is_logged_in: boolean;
+    is_cfd_password_modal_enabled: boolean;
+    is_cfd_success_dialog_enabled: boolean;
+    is_dxtrade_allowed: boolean;
+    platform: string;
+    has_cfd_error: boolean;
+    has_suspended_account: boolean;
+    landing_companies: LandingCompany;
+    mt5_login_list: TExtendedDetailsOfEachMT5Loginid[];
+    cfd_new_account: Mt5NewAccount;
+    setCFDSuccessDialog: (value: boolean) => void;
+    setMt5Error: (state: boolean, obj?: Error) => void;
+    submitMt5Password: (
+        values: TCFDPasswordFormValues & { server?: string },
+        actions: FormikActions<TCFDPasswordFormValues>
+    ) => void;
+    submitCFDPassword: (
+        values: TCFDPasswordFormValues & { platform?: string },
+        actions: FormikActions<TCFDPasswordFormValues>
+    ) => void;
```

Comment:
Can we reuse this fn type? It's used in more than 3 places.

Reply:
I have reused a similar one: `(values: TCFDPasswordFormValues, actions: FormikActions<TCFDPasswordFormValues>) => void;`, which is used multiple times, because it seems like the submitCFDPassword type is used once.
thank you @balakrishna-binary  😊